networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-load-balancer-allowed-source-ranges.adoc

```diff
@@ -0,0 +1,16 @@
+:_content-type: ASSEMBLY
+[id="configuring-ingress-cluster-traffic-lb-allowed-source-ranges"]
+= Configuring ingress cluster traffic using Load Balancer allowed source ranges
+include::_attributes/common-attributes.adoc[]
+:context: configuring-ingress-cluster-traffic-lb-allowed-source-ranges
+
+toc::[]
+
+You can enable and configure the `spec.endpointPublishingStrategy.loadBalancer.allowedSourceRanges` field. You can specify a list of IP address ranges for the `IngressController`. This restricts access to the Load Balancer service when the `endpointPublishingStrategy` is `LoadBalancerService`.
+
+include::modules/nw-configuring-lb-allowed-source-ranges.adoc[leveloffset=+1]
+include::modules/nw-configuring-lb-allowed-source-ranges-migration.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+== Additional resources
+* See xref:../../updating/index.adoc#index[Updating your cluster].
```

Comment:
I'd recommend removing "See" and the period, and just leaving the xref stand alone. I think this is the preferred method for additional resources per jupiter specifications.

Reply:
Addressed